src/tactic/ui/panel/tile_layout_wdg.py

```diff
@@ -1694,30 +1718,32 @@ def get_scale_wdg(my):
             var src_top = bvr.src_el.getParent(".spt_tile_top");
             var src_code = src_top.getAttribute("spt_search_code");
             var parent_code = dst_top.getAttribute("spt_search_code");
-            if (parent_code == src_code) {
-                spt.notify.show_message("Cannot add a collection to itself");
-                return;
+            
+            if (parent_code != src_code){
+                var data = {
+                    parent_code: parent_code,
+                    search_code: src_code
+                };
+                var sobject = server.get_unique_sobject(collection_type, data);
```

Comment:
why are we asking the server for this sobject and don't make use of it?. I would remove it if it's not used.